.github/workflows/ci.yml

```diff
@@ -49,6 +50,12 @@ jobs:
             profile: "conda"
           - isMaster: false
             profile: "singularity"
+          - profile: "conda"
```

Comment:
If these workflows can't run with conda, I assume the whole pipeline can also be not run with conda when using bwameth. Is this documented somewhere?

Reply:
actually the pipeline can be run using conda. It seems there is only an issue when running conda on CI for pipeline tests. see [this issue](https://github.com/nf-core/modules/issues/6664)